spec/integration_spec.rb

```diff
@@ -369,6 +361,90 @@ def parse_html(html)
 
       expect(expected_result).to eq(actual_result)
     end
+
+    it 'groups non-inlineable media queries in the head by default' do
+      document = Roadie::Document.new <<-HTML
+        <html>
+          <head>
+            <link rel="stylesheet" href="/style.css">
+          </head>
+          <body>
+            <div class="colorful"></div>
+          </body>
+        </html>
+      HTML
+
+      document.asset_providers = TestProvider.new(
+        "/style.css" => <<-CSS
+          .colorful { color: green; }
+          @media screen and (max-width 600px) {
+            .colorful { color: red; width: 600px; }
+          }
+          @media screen and (max-width 600px) {
+            .colorful-2 { color: red; width: 600px; }
+          }
+        CSS
+      )
+
+      result = parse_html document.transform
+
+      styles = result.at_css('html > head > style').text
+      expected_result = <<-CSS
+        @media screen and (max-width 600px) {
+          .colorful { color:red;width:600px }
+          .colorful-2 { color:red;width:600px }
+        }
+      CSS
+      expected_result = expected_result.gsub(/[\s]+/, ' ').strip
+      actual_result = styles.gsub(/[\s]+/, ' ').strip
+
+      expect(expected_result).to eq(actual_result)
+    end
+
+    describe 'if use_shared_media_queries is set to false' do
```

Comment:
Prefer `describe "when ..."`

`describe "when use_shared_media_queries is false"`